CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to status-go
```

Comment:
these are some standard guidelines we share across projects?

Reply:
I just copy / pasted / adapted from status-go repo, not sure if we have a standard CONTRIBUTING for new repos.